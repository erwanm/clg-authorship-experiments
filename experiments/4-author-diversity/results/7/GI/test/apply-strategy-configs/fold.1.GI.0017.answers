1903-us-tjoas 1854-hbs-smofl	0.012858
1895-wta-alae 1906-mt-cfma	0.196017
1891-fmc-ttf 1881-haj-fcbtp	0.507022
1872-haj-ptf 1906-rwc-ttolp	0.01335
1898-hj-ttm 1879-hj-h	0.001439
1874-wta-tcw 1900-haj-adoh	0.007511
1878-hj-te 1882-lma-ajsb-v6	0.015358
1883-hj-rh 1898-hj-itc	0.221772
1885-hj-gr 1910-us-sts	0.005222
1914-us-sm 1897-ewaoc-tdof	0.006678
1896-espw-cfal 1887-espw-jtf	0.166022
1887-espw-tgb 1900-espw-wtg	0.151729
1894-cdw-tgh 1907-fmc-tdr	0.407945
1900-fmc-itpotk 1905-fmc-fm	0.213817
1889-fmc-g 1898-haj-tyb	0.061613
1890-hj-ttm 1904-hj-tgb	0.015334
1878-hbs-pp 1870-hbs-tbc	0.151499
1901-us-km 1921-us-tbol	0
1883-hj-rh-v2 1872-wta-lb	0
1866-haj-ccc-ia 1871-hbs-mwai	0.00291
1880-hj-c 1885-hj-altof	0.352651
1898-fmc-ari-v2 1893-fmc-kl-v2	0
1918-rwc-tlg 1863-wta-isao	0.012297
1884-ab-sbts 1898-us-apom	0
1889-cfw-jl 1915-rwc-p	0.00836
1872-mt-ri 1895-haj-tdw	0.00824
1902-fmc-c 1883-fmc-tl	0.199555
1859-hbs-tmw 1887-fmc-pp	0.351524
1884-wta-Sac 1882-wta-aa	1
1871-haj-tt 1876-wta-vao	0.42155
1913-es-tm 1911-rwc-tgr	0.405673
1905-es-tpoe 1907-es-tlotg	0.012499
1899-haj-mmv 1866-haj-hf	0.014066
1891-fmc-do 1876-cdw-itl	0
1892-fmc-pg 1883-fmc-dc	0.400264
1917-ab-bn 1922-us-tcmc	0.019642
1889-cdw-sths 1869-espw-mwag	0.000988
1902-hj-twotd 1896-hj-tsop	0.007541
1888-haj-bb 1899-haj-ra	0.563805
1913-ew-tcotc 1868-haj-rd	0.007034
1868-espw-tga 1919-rwc-tsos	0.20965
1900-haj-ainy 1919-ew-m	0.001165
1881-mt-tpatp 1899-haj-aoatb	0.007242
1891-fmc-k 1893-fmc-tr	0.01245
1881-hj-tpoal 1890-fmc-acmr	0.118044
1869-haj-mtmb 1903-haj-bba-ph	0.612848
1885-fmc-atoalp 1905-fmc-wso	1
1868-wta-mob 1870-wta-faf	0.504304
1871-hbs-pawt 1904-rwc-isotu	0.216659
1875-cfw-cn 1864-lma-m	0.009143
1914-mh+-mhce-nf 1908-fmc-tp	0.005052
1893-fmc-md 1895-fmc-ajs	0.861351
1864-wta-tsb 1883-wta-sh	0.007359
1900-rwc-taoamm 1910-rwc-tgm	0.015663
1903-fmc-thor 1887-fmc-s	0.307625
1884-mt-taohf 1874-mt-lotm	0.510328
1907-espw-tldup 1871-espw-tsp	0
1866-espw-gcj 1882-haj-bn	0.405718
1883-haj-tbb 1888-es-ttatv	0.19709
1908-fmc-s 1908-fmc-tws	0.808269
1889-wta-wtel 1911-rwc-tcl	0
1904-rwc-sm 1886-hj-pc	0.003113
1918-us-jh 1897-ew-tdof	0.205154
1872-lma-ss 1867-lma-tagomtt	0.009657
1910-ew-tomag 1889-es-atg	0.004182
1889-espw-tsfi 1895-espw-tsojc	0.309835
1905-haj-fftf 1870-haj-rar	0.458024
1882-espw-dz 1909-mt-isd	0.019944
1907-ew-tfott 1896-hj-toh	0.609411
1868-wta-dad 1885-wta-sts	0.899375
1909-rwc-tdm 1897-ab-tdohy	0.012681
1891-haj-tetb 1868-haj-faf	0.805037
1893-haj-dtn 1866-haj-tcw	0.015639
1869-wta-dtr 1865-wta-waw	0.009271
1891-cdw-oi 1873-hbs-wish	0.055366
1879-hj-dm 1903-hj-ta	0.003246
1874-cfw-tac 1882-fmc-mi	0.298047
1893-espw-dm 1896-ab-boat	0.005482
1896-espw-tsasa 1894-espw-asl	0.009182
1879-haj-tym 1869-haj-pjb	1
1920-us-1tsoap 1913-us-dg	0.003814
1917-es-ow-sh 1873-hbs-pl	0.015384
1855-wta-aa 1873-wta-tyc	0.018577
1879-haj-ttb 1894-fmc-wg	0
1898-fmc-vc 1895-fmc-t	0.509243
1867-espw-gyatgc 1901-espw-a	0.007876
1896-fmc-c 1885-fmc-zamc	0.556135
1855-hbs-tmf 1866-espw-gb	1
1897-fmc-aroy 1884-fmc-aap	0.411759
1883-espw-btg 1911-hj-to	0.054977
1862-hbs-tpooi 1875-hbs-waon	0.406486
1883-fmc-ars 1885-fmc-tcotk	0.160742
1886-hbs-tsw 1854-hbs-smofl-v2	0.40343
1903-ew-ivatg 1878-cdw-hikab	0.403771
1858-wta-pap 1897-haj-fof	0.606969
1902-hj-twotd-v2 1888-hj-tr	0.209541
1871-hbs-ofs 1888-fmc-si	0.009238
1907-ab-rm 1901-hj-tsf	0.065283
1882-haj-ffbts 1893-haj-ianwatgfoa	0
1908-rwc-tfl 1910-rwc-ap	0.507608
