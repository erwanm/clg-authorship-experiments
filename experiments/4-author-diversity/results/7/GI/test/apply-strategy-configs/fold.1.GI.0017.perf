0.388920	0.648200	0.600000
