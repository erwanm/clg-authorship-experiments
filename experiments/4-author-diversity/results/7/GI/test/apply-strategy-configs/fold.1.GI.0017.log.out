apply-config.sh: applying config '/tmp/expe4/7/process/selected-meta-configs/fold.1.strategy-as-meta.GI.0017.model/strategy-configs/fold.1.GI.0017' to cases '/home/moreaue/dump/expe4//results/7/GI/test/apply-strategy-configs/only-cases.tmp.VgNcCk0e'; results written to '/home/moreaue/dump/expe4//results/7/GI/test/apply-strategy-configs/fold.1.GI.0017'
apply-config.sh: done.
